src/main/java/com/amazon/ion/impl/IonCursorBinary.java

```diff
@@ -1645,6 +1657,13 @@ IonTypeID typeIdFor(int length) {
                 return TYPE_IDS_1_1[OpCodes.SYMBOL_ADDRESS_MANY_BYTES & SINGLE_BYTE_MASK];
             }
         },
+        SYSTEM_SYMBOL_ID {
+            @Override
+            IonTypeID typeIdFor(int length) {
+                // if (length > 1) throw new IllegalStateException("System Symbols always have a length of 1");
```

Comment:
Should this validation be un-commented, or removed?

Reply:
Removed.